### PR TITLE
Fixes violations of and reenables the HAML LeadingCommentSpace check.

### PR DESCRIPTION
**Pull request by ashercodeorg**

### CODE REVIEW

**Comment by ashercodeorg** [comment]
Tanya: A lot of these seem to be commented out code. Ideally, these code snippets would be deleted (if no longer relevant) or documented with an explanation as to why the code is commented and/or why it might be uncommented.

File: pegasus/sites.v3/code.org/layouts/wide.haml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 .wrapper
   -if header['theme'] == 'responsive_wide'
     #desktop-header.desktop-feature
-      -#= view :header
+      -# = view :header
```
Comment (tanyaparker) [inline pegasus/sites.v3/code.org/layouts/wide.haml:4]: @breville any reason to keep this here or can I send you a PR with the reverse logic?

**Comment by tanyaparker** [comment]
lgtm

I haven't touched all of those files, but for the ones that I know about I'll remove or add a comment.

File: pegasus/sites.v3/code.org/layouts/wide.haml (hunk 1 above)
Comment (breville) [inline pegasus/sites.v3/code.org/layouts/wide.haml:4]: thanks for doing the cleanup in the other PR

File: pegasus/sites.v3/ar.code.org/views/top_quotes.haml [hunk 2]
```diff
@@ -13,8 +13,9 @@
...
         =value[:quote_t]
 
--#links{:style=>'clear:both;'}
-  = view :share_buttons, facebook:facebook, twitter:twitter)
-  .all-quotes-link
-    %a{:href=>'http://code.org/quotes'}
-      Mirá todas las citas (en inglés)
\ No newline at end of file
+-#
+  links{:style=>'clear:both;'}
```
Comment (joshlory) [inline pegasus/sites.v3/ar.code.org/views/top_quotes.haml]: Was this change intended?  Uncommenting `links{:style=>'clear:both;'}`.

Comment (joshlory) [inline pegasus/sites.v3/ar.code.org/views/top_quotes.haml]: Never mind, I see now.

Comment (tanyaparker) [inline pegasus/sites.v3/ar.code.org/views/top_quotes.haml:17]: Yeah, we hide them on this site. My future PR removes it.

https://github.com/code-dot-org/code-dot-org/pull/7393
